plugins/json_importer.cc

```diff
@@ -211,6 +250,14 @@ struct JSONImporter: public Procedure {
                 return handleError(exc.what(), actualLineNum, string(line, lineLength));
             }
 
+            if (useWhere) {
+                JsonScope jsonScope(server, expr);
+                auto whereBound = config.where->bind(jsonScope);
```

Comment:
Binding should happen outside of the onLine handler.  Otherwise it will be horrendously slow.